apps/settings/js/panels/languages/languages.js

```diff
@@ -56,7 +56,7 @@ define(function(require) {
     },
     showMoreLanguages: function() {
       SettingsCache.getSettings(function(result) {
-        var version = result['deviceinfo.os'];
+        var version = result['langpack.channel'];
```

Comment:
The value is controlled by https://github.com/mozilla-b2g/gaia/blob/899860aa2f58cb2597405cd3f8572a938e7ca384/build/config/common-settings.json#L134